extensions/libraries/redcore/database/sqlparser/sqlparser.php

```diff
@@ -43,20 +43,43 @@
  * @author greenlion@gmail.com
```

Comment:
@thongredweb there are a ton of changes in sqlparser.php but they are mostly code style changes right? Did you changed anything relevant here?